### ISSUE TITLE
No load on server startup

### ISSUE THREAD

**Issue by Momonche** [spigot-gillesm/RPGChest-Premium#8]
Hello Gilles !

When I start my server the plugin fail to load : 

```
[18:44:51 INFO]: [RPGChest] Enabling RPGChest v1.9.0.2
[18:44:51 INFO]: [RPG Chest] |x|-===MythicMobs not detected===-|x|
[18:44:51 INFO]: [RPG Chest] |x|-===SkillAPI not detected===-|x|
[18:44:51 INFO]: [RPG Chest] |x|-===MMOItems not detected===-|x|
[18:44:51 INFO]: [RPG Chest] |x|-===RPGRegen not detected===-|x|
[18:44:51 INFO]: [RPG Chest] |v|-===Vault dependency detected===-|v|
[18:44:51 ERROR]: [RPGChest] Plugin RPGChest v1.9.0.2 has failed to register events for class me.gilles_m.RPGChestPremium.goto. .coM6 because io/lumine/xikage/mythicmobs/api/bukkit/events/MythicMobDeathEvent does not exist.
[18:44:53 INFO]: [RPG Chest] Checking for updates in language.yml...
[18:44:53 INFO]: [RPG Chest] Done!
[18:44:54 ERROR]: Error occurred while enabling RPGChest v1.9.0.2 (Is it up to date?)
java.lang.NullPointerException: null
at org.bukkit.Location.getChunk(Location.java:123) ~[patched_1.15.1.jar:git-Paper-50]
at me.gilles_m.RPGChestPremium.goto. .Com7.int(SourceFile:152) ~[?:?]
at me.gilles_m.RPGChestPremium.goto. .Com7.<init>(SourceFile:99) ~[?:?]
at me.gilles_m.RPGChestPremium.goto. .Com7.new(SourceFile:199) ~[?:?]
at me.gilles_m.RPGChestPremium.RPGChest.onEnable(SourceFile:139) ~[?:?]
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.15.1.jar:git-Paper-50]
at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338) ~[patched_1.15.1.jar:git-Paper-50]
at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:420) ~[patched_1.15.1.jar:git-Paper-50]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:468) ~[patched_1.15.1.jar:git-Paper-50]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:382) ~[patched_1.15.1.jar:git-Paper-50]
at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:481) ~[patched_1.15.1.jar:git-Paper-50]
at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:290) ~[patched_1.15.1.jar:git-Paper-50]
at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:884) ~[patched_1.15.1.jar:git-Paper-50]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
[18:44:54 INFO]: [RPGChest] Disabling RPGChest v1.9.0.2
```

BUT when the server is ON, I use PlugMan to reload the plugin and it works : 

```
[18:46:01 INFO]: MOMO3915 issued server command: /plugman reload RPGChest
[18:46:04 INFO]: [RPGChest] Enabling RPGChest v1.9.0.2
[18:46:04 INFO]: [RPG Chest] |x|-===MythicMobs not detected===-|x|
[18:46:04 INFO]: [RPG Chest] |x|-===SkillAPI not detected===-|x|
[18:46:04 INFO]: [RPG Chest] |x|-===MMOItems not detected===-|x|
[18:46:04 INFO]: [RPG Chest] |x|-===RPGRegen not detected===-|x|
[18:46:04 INFO]: [RPG Chest] |v|-===Vault dependency detected===-|v|
[18:46:04 ERROR]: [RPGChest] Plugin RPGChest v1.9.0.2 has failed to register events for class me.gilles_m.RPGChestPremium.goto. .coM6 because io/lumine/xikage/mythicmobs/api/bukkit/events/MythicMobDeathEvent does not exist.
[18:46:05 INFO]: [RPG Chest] Checking for updates in language.yml...
[18:46:05 INFO]: [RPG Chest] Done!
[18:46:06 INFO]: [RPG Chest] Checking for updates...
[18:46:06 INFO]: [RPG Chest] Using latest version
[18:46:06 INFO]: [RPG Chest] Enabled
```

I can see "at org.bukkit.Location.getChunk(Location.java:123)"
I think it because some of my RPGChest are in a world monitoring by Multiverse-core ? (not sure)

Server git-Paper-50 (MC 1.15.1)
RPGChest 1.9.0.2
PlugMan 2.1.6
Multiverse-Core 4.1.0-b775

**Comment by spigot-gillesm** [comment]
Hello, I'll dig in the code to find out why it's crashing. Can you tell me if it always crashes on server load or if it sometimes loads properly?

**Comment by Momonche** [comment]
I test with 20~30 restart and the plugin crash every time, so I can say it always append

**Comment by Momonche** [comment]
I just test one time with RPGChest 1.9.0.3 and still have the problem

I'll test to restart my server 20/30 times for more informations

Edit: I just finish to restart my server 20 times and there is ALWAYS the issue

**Comment by Momonche** [comment]
And I can see, this error disappear from 1.9.0.2 : 

```
[18:44:51 ERROR]: [RPGChest] Plugin RPGChest v1.9.0.2 has failed to register events for class me.gilles_m.RPGChestPremium.goto. .coM6 because io/lumine/xikage/mythicmobs/api/bukkit/events/MythicMobDeathEvent does not exist.

```

Did you do something about that ?

**Comment by etu40627** [comment]
Oh I thought the event loading error was the main problem sorry I’ll see what I can do about the other issue as soon as I can
Sorry for the inconvenience

**Comment by spigot-gillesm** [comment]
Oh I thought the event loading error was the main problem sorry I’ll see what I can do about the other issue as soon as I can
Sorry for the inconvenience

**Comment by Momonche** [comment]
Fix on RPGChest 1.9.2.0

Thank you !
